khmer/_khmer.cc

```diff
@@ -401,6 +401,7 @@ _ReadParser_iternext( PyObject * self )
     khmer_ReadParser_Object * myself  = (khmer_ReadParser_Object *)self;
     IParser *       parser  = myself->parser;
 
+    // cppcheck-suppress unreadVariable
```

Comment:
If we replace this with a `bool stop_iteration;` declaration, we get an unused variable warning.